test/check-application

```diff
@@ -58,6 +57,60 @@ class TestApplication(testlib.MachineCase):
         b.wait_present("#containers-containers")
         b.wait_in_text("#containers-containers", "No running containers")
 
+        # delete container busybox:latest from container listing toggle
+        b.click("#containers-containers-filter button")
+        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='all'] a")
+        b.wait_visible("#containers-containers")
+        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+
+        # click Start container button from container listing toggle
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr div.dropdown')
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr:contains("Start")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr:contains("Start")')
+
+        # delete container from listing toggle
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        with b.wait_timeout(20):
```

Comment:
Please keep the standard 60 second timeout (also below). It should not hurt, and on our loaded CI infra things are really slow sometimes.